### PR TITLE
Article about OpenCage sponsorship of metaCPAN

### PR DESCRIPTION
**Pull request by freyfogle**
As requested by @oalders

### CODE REVIEW

File: content/article/why-we-sponsor-metacpan-opencage.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+  {
+    "title"       : "Why we sponsor MetaCPAN: OpenCage",
+    "authors"     : ["ed-freyfogle"],
+    "date"        : "2024-07-30T10:35:43",
+    "tags"        : ["opencage", "metacpan", "sponsorship", "geocoding"],
+    "draft"       : true,
```
Comment (oalders) [inline content/article/why-we-sponsor-metacpan-opencage.md:7]: We may as well set this to `false`, as we'll want to publish right after merge anyway.

**Comment by oalders** [comment]
Thanks, @freyfogle! If you're able to create this PR directly from `perladvent:master` (rather than your fork) and set `draft: false`, then we'll get a live preview URL attached to the PR.

**Comment by freyfogle** [comment]
HI, I don't know what "create this PR directly from perladvent:master (rather than your fork)" means. 
How do I do that?

**Comment by freyfogle** [comment]
ok, I think I get what you mean, please see PR #383

**Pull request closed without merging**